packages/contracts/contracts/core/DAO.sol

```diff
@@ -28,6 +28,7 @@ contract DAO is IDAO, Initializable, UUPSUpgradeable, ACL, ERC1271, AdaptiveERC1
     bytes32 public constant EXEC_ROLE = keccak256("EXEC_ROLE");
     bytes32 public constant WITHDRAW_ROLE = keccak256("WITHDRAW_ROLE");
     bytes32 public constant SET_SIGNATURE_VALIDATOR_ROLE = keccak256("SET_SIGNATURE_VALIDATOR_ROLE");
+    bytes32 public constant MODIFY_FORWARDER = keccak256("MODIFY_TRUSTED_FORWARDER");
```

Comment:
I think we should name the constant ``MODIFY_TRUSTED_FORWARDER`` to keep consistency.. wdyt?

Reply:
Yes, I thought the same.